File: tournaments/quadraticEquation/quadraticEquation.cs
double[] quadraticEquation(int a, int b, int c) {
    int discriminant =  b*b - 4 * a * c;
    if (0 > discriminant) {
        return new double[0];
    }
    
    if (0 == discriminant) {
        return new double[] {(-b / (2.0 * a))};
    }
    
    var result = new List<double>();
    result.Add((-b - Math.Sqrt(discriminant)) / (2.0 * a));
    result.Add((-b + Math.Sqrt(discriminant)) / (2.0 * a));
    
    if (result[0] > result[1]) {
        var temp = result[1];
        result[1] = result[0];
        result[0] = temp;
    }
    
    return result.ToArray();
}

File: tournaments/quadraticEquation/quadraticEquation.js
function quadraticEquation(a, b, c) {
    const discriminant =  b ** 2 - 4 * a * c;

    if (0 > discriminant) {
         return [];
    }

    if (!discriminant) {
        return [-b / (2 * a)];
    }

    const result = new Array(2);
    result[0] = ((-b - Math.sqrt(discriminant)) / (2 * a));
    result[1] = ((-b + Math.sqrt(discriminant)) / (2 * a));

    if (result[0] > result[1]) {
        [result[0], result[1]] = [result[1], result[0]];
    }

    return result;
}
